luwak/src/main/java/uk/co/flax/luwak/presearcher/TermFilteredPresearcher.java

```diff
@@ -132,7 +135,7 @@ public Query build() {
 
     public static final FieldType QUERYFIELDTYPE;
     static {
-        QUERYFIELDTYPE = new FieldType(TextField.TYPE_STORED);
+        QUERYFIELDTYPE = new FieldType(TextField.TYPE_NOT_STORED);
         QUERYFIELDTYPE.setIndexOptions(IndexOptions.DOCS_AND_FREQS_AND_POSITIONS_AND_OFFSETS);
```

Comment:
We don't need offsets anymore, but I think it's worth keeping positions in.  I doubt it makes much difference at query-index time, and it makes debugging more difficult if you need to enable positions and then re-add everything.